providers/ms365/resources/ms365.lr

```diff
@@ -28,20 +30,30 @@ microsoft {
   tenantDomainName() string
 }
 
-// Microsoft organization
-private microsoft.organization @defaults("displayName") {
+//  Microsoft Entra tenant
+microsoft.tenant @defaults("name") {
```

Comment:
@tas50 the resource is public now so it can be called directly

Reply:
LGTM then